boards/wch/ch32v003evt/wch_ch32v003evt_defconfig

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2024 Michael Hope
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_CLOCK_CONTROL=y
```

Comment:
UART/console required

Reply:
Do you prefer to get the USART driver merged as part of this patch series (this PR)?

Reply:
```
$ cat boards/wch/ch32v003evt/wch_ch32v003evt_defconfig
# Copyright (c) 2024 Michael Hope
# SPDX-License-Identifier: Apache-2.0

CONFIG_CLOCK_CONTROL=y
CONFIG_PINCTRL=y
CONFIG_SERIAL=y
```

I have added the UART driver in a separate commit in this PR now -  thanks!